### PR TITLE
[AAE-10125] Storybook stories for Comments component

### PR DESCRIPTION
**Pull request by Venrofi**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/AAE-10125

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/mock/comment-process-service.mock.ts [hunk 1]
```diff
@@ -15,28 +15,54 @@
...
-    lastName: 'Adams',
-    email: 'wilbur@app.com'
-})});
+import { Injectable } from '@angular/core';
+import { Observable, from, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { CommentModel, UserProcessModel } from '../models';
+import { CommentProcessServiceInterface } from '../services/comment-process.service.interface';
+import { testUser, fakeUser1 } from '../mock/comment-content.mock';
+@Injectable()
```
Comment (tomgny) [inline lib/core/mock/comment-process-service.mock.ts:24]: missing empty line

**Review by tomgny — CHANGES_REQUESTED** [review]
Please modify comments module to import/export only necessary modules (e.g. remove MaterialModule and add only those modules that are needed)
https://alfresco.atlassian.net/browse/HXENG-74

File: lib/core/mock/comment-process-service.mock.ts (hunk 1 above)
Comment (Venrofi) [reply lib/core/mock/comment-process-service.mock.ts:24]: Fixed

File: lib/core/comments/comments.module.ts [hunk 2]
```diff
@@ -34,9 +38,13 @@ import { CommentsComponent } from './comments.component';
         DataTableModule,
         FormsModule,
         ReactiveFormsModule,
```
Comment (tomgny) [inline lib/core/comments/comments.module.ts:40]: Can you verify if we need these modules? I'm not sure that we need them

- `ReactiveFormsModule`
- `DataTableModule`
- `DataColumnModule`

Comment (Venrofi) [reply lib/core/comments/comments.module.ts:40]: The only module that is necessary is the **FormsModule**. Without it, the _mat-form-field_ throws an error. I'll delete the rest 😄
image

**Review by tomgny — APPROVED** [review]

**Pull request merged**